<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RendezVousDialog</class>
 <widget class="QDialog" name="RendezVousDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Rendez-vous</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxForm">
     <property name="title">
      <string>Formulaire Rendez-vous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPatient">
        <property name="text">
         <string>Patient :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxPatient"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMedecin">
        <property name="text">
         <string>Médecin :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxMedecin"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelHeure">
        <property name="text">
         <string>Heure :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelMotif">
        <property name="text">
         <string>Motif :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="textEditMotif"/>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <!-- Bouton Ajouter avec icône -->
        <item>
         <widget class="QPushButton" name="btnAjouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/ajouter.png</normaloff>
           </iconset>
          </property>
         </widget>
        </item>
        <!-- Bouton Supprimer avec icône -->
        <item>
         <widget class="QPushButton" name="btnSupprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/supprimer.png</normaloff>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <!-- Table des Rendez-vous -->
   <item>
    <widget class="QTableWidget" name="tableWidgetRendezVous">
     <property name="columnCount">
      <number>6</number>
     </property>
     <property name="horizontalHeaderLabels" stdset="0">
      <stringlist>
       <string>ID</string>
       <string>Patient</string>
       <string>Médecin</string>
       <string>Date</string>
       <string>Heure</string>
       <string>Motif</string>
      </stringlist>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
